### ISSUE TITLE
Missing logs on sentences from June 3 to June 10, 2017

### ISSUE THREAD

**Issue by soliloquist-tatoeba** [Tatoeba/tatoeba2#1921]
https://tatoeba.org/eng/sentences/show/6129089

Ivanovb and I had left some comments on the Turkish sentence above. After Gulo_Luscus' correction, the logs on the sentence page showing the previous version of the sentence were gone.

I had reported a similar issue on #1865, but this time, it has nothing to do with deleted users. The sentence was created by duran.

**Comment by trang** [comment]
The logs were not there in the first place. This is an issue resulting from the the [crash in 2017](https://blog.tatoeba.org/2017/06/tatoeba-is-back.html):

> Contributions made after June 3 don't have logs.

 We could restore sentences and links up until June 10th, but we could restore the logs only up until June 3rd. So any sentence between June 3rd and June 10th don't have logs. Sentence nº6129089 is one of them.

Several thousands of sentences are affected by this. I suppose we could create logs with an arbitrary "added" date (June 10th for instance). This would at least ensure that we know the previous version of the sentence if it ever gets modified.

**Comment by soliloquist-tatoeba** [comment]
I didn't notice anything unusual when I was leaving comments on the sentence, but since you crosschecked the sentence number with the crash records, that must be the reason. Thanks.

I agree that it would be better to add logs to such sentences showing their original versions in case they are edited.

**Comment by trang** [comment]
Actually, we can use the `created` date of the sentence to reconstruct the logs. So no need for an arbitrary date. The only thing that will be arbitrary is the contributor information. We can set it to the current owner even though technically, it could have been unadopted by the original contributor, then adopted by someone else.

**Comment by soliloquist-tatoeba** [comment]
I don't think showing creation dates and original contributors are very important as long as the original/previous versions of texts are shown.

There are a lot of sentences without such information.

https://tatoeba.org/eng/sentences/show/2350 

https://tatoeba.org/eng/sentences/search?query=&from=eng&to=none&orphans=yes&unapproved=no&user=&tags=&list=&has_audio=&trans_filter=limit&trans_to=und&trans_link=&trans_user=&trans_orphan=&trans_unapproved=&trans_has_audio=&sort=created&sort_reverse=yes

**Comment by trang** [comment]
The logs for the following sentences were missing: https://gist.github.com/trang/f44916ec69f5fed0ec76d7aee969a6f7

I reconstructed the logs for most of them.

The ones where I did not reconstruct the logs were sentences that were never restored ([437 sentences](https://gist.github.com/trang/3fff77373b66043ea968a9330c8501ae)). They belonged to users whose account have been completely lost in the crash. I won't be restoring them.
